### PR TITLE
Update RUM docs according to new pricing plans

### PR DESCRIPTION
**Pull request by bcaudan**
### What does this PR do?

Update RUM docs according to new pricing plans: 

- Update snippets with new init parameters
- Use new plans names

### Motivation

New RUM pricing plans.

### Additional Notes

Related Browser page updates: https://github.com/DataDog/browser-sdk/pull/1745

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by liywjl — APPROVED** [review]

**Comment by maycmlee** [comment]
Created DOCS-4137 for editorial review.

File: content/en/real_user_monitoring/guide/sampling-browser-plans.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+This feature requires the Datadog Browser SDK v3.0.0+.
+
+<blockquote class="alert alert-info">
+The Datadog Browser SDK v4.20.0 introduces the <code>sessionReplaySampleRate</code> initialization parameter, deprecating the <code>premiumSampleRate</code> and <code>replaySampleRate</code> initialization parameter.
+</blockquote>
+
+When a session is created, RUM tracks it as either:
+
+- [**Browser RUM**][2]: Sessions, views, actions, resources, long tasks and errors are collected. Calls to `startSessionReplayRecording()` are ignored.
```
Comment (apigirl) [inline content/en/real_user_monitoring/guide/sampling-browser-plans.md:29]:
```suggestion
- [**Browser RUM**][2]: Sessions, views, actions, resources, long tasks, and errors are collected. Calls to `startSessionReplayRecording()` are ignored.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ The RUM Browser SDK is [open source][1] and leverages the open source [rrweb][2]
 
 ## Session Replay recorder
 
-The Session Replay recorder is part of the RUM Browser SDK. The recorder takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with these events' timestamps. 
+The Session Replay recorder is part of the RUM Browser SDK. The recorder takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with these events' timestamps.
 
-Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 day retention policy as normal RUM sessions. 
+Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 days retention policy as normal RUM sessions.
```
Comment (apigirl) [inline content/en/real_user_monitoring/session_replay/_index.md:29]:
```suggestion
Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 day retention policy as normal RUM sessions.
```
I think the original here was more correct

**Review by apigirl — COMMENTED** [review]
just a few minor feedbacks

**Review by apigirl — APPROVED** [review]

**Pull request merged**